dwave/cloud/client.py

```diff
@@ -646,6 +647,21 @@ def _fetch_solvers(self, name=None):
 
         return solvers
 
+    def get_problem(self, id_):
+        """Retrieve a problem by id.
+        
+        Args:
+            id_ (str):
+                As returned by :attr:`Future.id`.
+
+        Returns:
+            :class:`Future`
+        
+        """
+        future = Future(None, id_)
+        self._load(future)
```

Comment:
Are you sure this will work when the problem is submitted, it's accepted (we have the ID), but it's still in `PENDING` or `IN_PROGRESS` state?

Reply:
I did check that, but it's been a few days so I will double check when I am making the name change etc